documentation/specs/Card.md

```diff
@@ -0,0 +1,194 @@
+
+### Features
+
+- Multiple border style variants
+- Background options
+- Deconstructed parts for composable complex cards
+
+### Risks and Challenges
+
+- Simple API with a variety of use cases
+- Ensuring the component can be composed to the variety of use cases
+
+### Prior Art
+
+- [Polaris `<AlphaCard />`](https://polaris.shopify.com/components/layout-and-structure/alpha-card)
+- [Paste `<Card />`](https://paste.twilio.design/components/card)
+
+---
+
+## Design
+
+A `Card` is semantically straightforward. Essentially serving as a container with preset styles. The complexity is surfacing a sufficiently advanced API that can accommodate the use cases without becoming overburdened.
+
+Architecture proposed is to surface a basic `<Card />` component with `variety` and `background` props while exposing the deconstructed parts to allow for maximum composability for complex use cases. The underlying pieces will be the `<Card.Container />` which houses the `variant` prop for determining the overall style of the card, and then a `<Card.Area />` which contains the inner container properties such as padding and the `background`.
+
+### API
+
+```ts
+type CardBackground = "primary" | "subdued";
+type CardVariant = "solid" | "outlined" | "danger" | "warning" | "success";
+
+type CardContainerProps = {
+  /**
+   * Card variant.
+   * @default solid
+   */
+  variant?: CardVariant;
+
+  /** Content of the card container. */
+  children: ReactNode;
+};
+
+type CardAreaProps = {
+  /** Background of the card area. By default, card backgrounds are transparent. */
+  background?: CardBackground;
+
+  /** Content of the card area. */
+  children: ReactNode;
+};
+
+type CardProps = CardContainerProps & CardAreaProps;
+```
+
+### Example Usage
+
+_Solid:_
+
+```tsx
+import { Card } from "@easypost/easy-ui/Card";
+
+function Component() {
+  return <Card>Content</Card>;
+}
+
+// same as:
+function Component() {
+  return <Card variant="solid">Content</Card>;
+}
+```
+
+_Outlined:_
+
+```tsx
+import { Card } from "@easypost/easy-ui/Card";
+
+function Component() {
+  return <Card variant="outlined">Content</Card>;
+}
+```
+
+_Flagged:_
+
+```tsx
+import { Card } from "@easypost/easy-ui/Card";
+
+function Component() {
+  // can be either "danger", "warning", "success"
+  return <Card variant="danger">Content</Card>;
+}
+```
+
+_Primary background:_
+
+```tsx
+import { Card } from "@easypost/easy-ui/Card";
+
+function Component() {
+  return <Card background="primary">Content</Card>;
+}
+```
+
+_Subdued background:_
+
+```tsx
+import { Card } from "@easypost/easy-ui/Card";
+
+function Component() {
+  return <Card background="subdued">Content</Card>;
+}
+```
+
+_Complex Card:_
+
+Card split into two columns using `HorizontalGrid` and deconstructed Card pieces:
+
+```tsx
+import { Card } from "@easypost/easy-ui/Card";
+
+function Component() {
+  return (
+    <Card.Container {...args}>
+      <HorizontalGrid columns={2}>
```

Comment:
**question**: do you envision the `HoriztonalGrid` to be a general purpose layout component?

Reply:
yeah it would be general purpose layout primitive, along with `VerticalStack`. both from Polaris. they both seem pretty solidly designed and useful.

i actually like a lot of layout components that polaris have, at least their new ones:

https://polaris.shopify.com/components/layout-and-structure

i imagine we may end up having most of them